t/app/controller/noise.t

```diff
@@ -61,7 +79,40 @@ FixMyStreet::override_config {
         is $report->detail, "Kind of noise: music\nNoise details: Details\n\nWhere is the noise coming from? residence\nNoise source: 100000333\n\nIs the noise happening now? Yes\nWhat days does the noise happen? monday, thursday\nWhat time does the noise happen? morning, evening\n";
         is $report->latitude, 53;
     };
-    subtest 'Report new noise, your address missing, source address unknown' => sub {
+    subtest 'Report new noise, your address missing, source address not a postcode' => sub {
+        $mech->get_ok('/noise');
+        $mech->submit_form_ok({ button => 'start' });
+        $mech->submit_form_ok({ with_fields => { existing => 0 } });
+        $mech->submit_form_ok({ with_fields => { name => "Test McTest", email => $user->email, phone => '01234 567890' } });
+        $mech->submit_form_ok({ with_fields => { best_time => [['day', 'evening'], 1], best_method => 'email' } });
+        $mech->submit_form_ok({ with_fields => { postcode => 'SW1A 1AA' } });
+        $mech->submit_form_ok({ with_fields => { address => 'missing' } });
+        $mech->submit_form_ok({ with_fields => { address_manual => 'My Address' } });
+        $mech->submit_form_ok({ with_fields => { kind => 'diy' } });
+        $mech->submit_form_ok({ with_fields => { where => 'residence', source_location => 'A street' } });
+        $mech->submit_form_ok({ with_fields => { latitude => 51.549239, longitude => -0.054106, radius => 'medium' } });
+        $mech->submit_form_ok({ with_fields => {
+            happening_now => 0,
+            happening_days => [['tuesday'], 1],
+            happening_time => [['morning','evening'], 1],
+        } });
+        $mech->submit_form_ok({ with_fields => { more_details => 'Details' } });
+        # Check going back skips the geocoding step
+        $mech->submit_form_ok({ form_number => 3, fields => { goto => 'where' } });
+        $mech->submit_form_ok({ with_fields => { where => 'residence', source_location => 'A street' } });
```

Comment:
This could be a location with multiple results, to show that that works (it currently does not), to be the reverse of the other test.